packages/context/test/acceptance/class-level-bindings.ts

```diff
@@ -171,6 +171,21 @@ describe('Context bindings - Injecting dependencies of classes', () => {
     expect(ctx.getSync('key')).to.equal('a-value');
   });
 
+  it('injects a nested property', async () => {
+    class TestComponent {
+      constructor(
+        @inject('config#test')
```

Comment:
Do we support `@inject('config#rest.length')`?

Reply:
Yes, we do. Support for deeply nested properties is covered by unit-tests, there is no need to duplicate such tests in the acceptance tests IMO.